### ISSUE TITLE
sapling zaddrs do not work in regtest

### ISSUE THREAD

**Issue by leto** [jl777/komodo#1247]
Test suite code such as `rpc.z_sendmany( taddr, recipients, minconf)`

generates:
```
JSONRPC error: Invalid parameter, unknown address format: zregtestsapling1mwmz8tlx3xvdq4zjwc9sfe0hsrzn440755h2329yxzygxpal472v3k0uwdeteepdj4z6wwa38un
```

IsValidPaymentAddress() returns false, which causes this exception to be thrown. Currently this blocks fully testing changes in my `dpowminconfs` branch

**Comment by leto** [comment]
FYI this no longer blocks merging `dpowconfs`, it's merged and I manually tested it

**Comment by jl777** [comment]
ok, so we can close this issue?

**Comment by leto** [comment]
@jl777 no, we still have a bug in regtest, because we didn't pull in some code that understands that regtest sapling addresses have a different bech32 prefix in regtest, which means we can't run a test for anything involving a zaddr in regtest currently
